sbws/util/stem.py

```diff
@@ -28,21 +27,25 @@ def closure_stream_event_listener(st):
             except (UnsatisfiableRequest, InvalidRequest) as e:
                 log.warning('Couldn\'t attach stream to circ %s: %s',
                             circ_id, e)
+            except Exception as e:
+                log.exception("Exception trying to get ns %s", e)
```

Comment:
I'm pretty sure this message should be something else. Maybe 'Unknown exception while trying to attach stream {st.id} to circ {circ_id}: {e}' (but with real Python)